eclipse-che/che-operator
# feat: Update memory for Plugin-Registry to 1536Mi

### What does this PR do?
When OpenVSX will be included in the plug-in registry
It'll require more memory to start.
With 1536Mi I didn't received Heap space/OOM/killed process

### Screenshot/screencast of this PR



### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/20549


### How to test this PR?
Deploy che and check more memory is assigned for plug-in registry container


Also you can check with the OpenVSX/plug-in registry instance
with the following patch
```bash
cat EOF << > /tmp/patch.yaml
spec:
  components:
    pluginRegistry:
      deployment:
        containers:
          - image: 'quay.io/fbenoit/che-plugin-registry:openvsx-20220729'
EOF

chectl server:deploy \
     --installer operator \
     --platform <PLATFORM_TO_DEPLOY> \
     --che-operator-image <CUSTOM_OPERATOR_IMAGE> \
     --che-operator-cr-patch-yaml /tmp/patch.yaml
```

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.

Change-Id: I125f1cbb5f6974f23d3b1942c3581f6a8d986046
Signed-off-by: Florent Benoit <fbenoit@redhat.com>
